src/icons/fileIcons.ts

```diff
@@ -2070,5 +2070,6 @@ export const fileIcons: FileIcons = {
         'steadybit.yaml',
       ],
     },
+    { name: 'chapel', fileExtensions: ['chpl']},
```

Comment:
Is the failure asking me to insert a blank space between the `]` and `}`?

Reply:
yes, I think so :)